src/modules/pages/components/Dashboard/Admin/Admin.jsx

```diff
@@ -43,11 +51,20 @@ type Props = {|
   /* eslint-disable-next-line react/no-unused-prop-types */
   path: string,
   user: User,
+  wallet: WalletObjectType,
 |};
 
-const Admin = ({ colonyClient, network, user }: Props) => {
-  const [visible, setVisible] = useState('AddAdmin');
-  if (!user || !user.admin || !user.admin[network.slug]) {
+const Admin = ({ colonyClient, network, user, wallet }: Props) => {
+  const [visible, setVisible] = useState('Accounts');
```

Comment:
I know we didn't do this before, but I feel like we should flow type the state here. Something along the lines of:

```suggestion
  const [visible, setVisible] = useState<'Accounts' | 'AddAdmin'>('Accounts');
```

Reply:
Yup, I seemed to have gotten lazy. Updated. :+1: